specification/_types/analysis/icu-plugin.ts

```diff
@@ -51,16 +51,16 @@ export class IcuFoldingTokenFilter extends TokenFilterBase {
 export class IcuCollationTokenFilter extends TokenFilterBase {
```

Comment:
Just noticed that the `IcuCollationTokenFilter` field names do not seem to be correct. Let`s fix them as well while we are on it...